src/main/java/se/diabol/jenkins/pipeline/DeliveryPipelineView.java

```diff
@@ -117,13 +112,20 @@
 
     private transient String error;
 
+    private static Random generator = new Random();
+    private String id = null;
+    private String viewUrl = null;
+    private boolean isPortletView = false;
```

Comment:
@pskumar448 How come it was not possible to extract those fields? I'm a bit restrictive adding fields if it is not obvious where they belong (e.g. id or a random generator).